sfge/src/main/java/com/salesforce/graph/vertex/MethodVertex.java

```diff
@@ -95,6 +95,12 @@ public LazyVertexList<AnnotationVertex> _getAnnotations() {
                                 .by(Schema.CHILD_INDEX, Order.asc));
     }
 
+    /** Generate a unique key associated with this method. */
+    public String generateUniqueKey() {
+        // Format of keys is "definingType#methodName@beginLine".
+        return this.getDefiningType() + "#" + this.getName() + "@" + this.getBeginLine();
```

Comment:
Generally curious - how do we store "definingType" in the case of an Inner class? Would the unique key look different for two methods with the same name, one in an inner class and another in an outer class, where both the inner class and outer class have the same name?

Reply:
For inner classes, the defining type is `outertype.innertype`, so there's no collision concern there.